src/lib/components/vegachart/VegaChart.component.js

```diff
@@ -1,7 +1,8 @@
 //@flow
```

Comment:
Thanks for the fix. Did you check if it fixes the issue in MetalK8s UI. FYI, you can simply take this specific commit and replace the package.json in MetalK8s  "@scality/core-ui": "github:scality/core-ui.git#3a7cd45",

In terms of the format of the commit message, I would put the ticket number in the last end. 
> VegaChart: Add cleanup function in effect to avoid memory leaking 
>  
> (Here you can describe more what's the issue/ how to solve the issue.)
> 
> Refs: #214